calico/felix/endpoint.py

```diff
@@ -437,42 +437,43 @@ def _get_endpoint_rules(endpoint_id, suffix, ip_version, local_ips, mac,
 
     # First build the chain that manages packets to the interface.
     # Common chain prefixes.
-    to_chain = []
+    to_chain = [
+        "--append %s --jump MARK --set-mark 0" % to_chain_name
+    ]
 
     # Jump to each profile in turn.  The profile will do one of the
     # following:
     # * DROP the packet; in which case we won't see it again.
-    # * RETURN without MARKing the packet; indicates that the packet was
-    #   accepted.
-    # * RETURN with a mark on the packet, indicates reaching the end
-    #   of the chain.
+    # * RETURN the packet with MARK==1, indicating it accepted the packet.
+    # * RETURN the packet with MARK==0, indicating it did not match the packet.
```

Comment:
I can't easily review this change, because - it appears - we don't have an overall comment that presents the iptables design as a whole.

For example, the change right begs the question "why not just ACCEPT, instead of RETURN with MARK==1".

Could you add, somewhere, an overall comment that presents the whole design?